api/envoy/config/filter/http/admission_control/v2alpha/admission_control.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.admission_control.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+import "envoy/type/grpc/grpc_status.proto";
+import "envoy/type/http_status.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "google/rpc/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.admission_control.v2alpha";
+option java_outer_classname = "AdmissionControlProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.admission_control.v3alpha";
+
+// [#protodoc-title: Admission Control]
+// Admission Control :ref:`configuration overview
+// <config_http_filters_admission_control>`.
+// [#extension: envoy.filters.http.admission_control]
+
+message AdmissionControl {
+  // Default method of specifying what constitutes a successful request. All status codes that
+  // indicate a successful request must be explicitly specified if not relying on the default
+  // values.
+  message DefaultSuccessCriteria {
+    // HTTP status codes to consider as request successes. If unspecified, defaults to "200".
```

Comment:
I think 200 is probably not the right default, given the prevalence of 204, etc. I would probably say any status code in the range 200-299 is considered success?

Reply:
Good point. I should implement some way to specify a "status range" or something like that.